src/DeviceInterfaceValues.tsx

```diff
@@ -394,6 +413,17 @@ const SendInterfaceDataModal = ({
                   <option value="true">true</option>
                   <option value="false">false</option>
                 </Form.Select>
+              ) : selectedMapping.type === 'datetime' ? (
+                <DatePicker
+                  selected={selectedDate}
+                  onChange={(date: Date | null) => handleValuesChange(date)}
+                  maxDate={maxDate}
```

Comment:
let's not use a `maxDate` maybe the user wants to send a future timestamp to book an appointment for next week